analyzers/src/SonarAnalyzer.CSharp/Rules/ForLoopCounterChanged.cs

```diff
@@ -111,5 +115,15 @@ private static IEnumerable<ISymbol> LoopCounters(ForStatementSyntax node, Semant
             node.DescendantNodesAndSelf()
                 .Where(n => SideEffectExpressions.Any(s => s.Kinds.Any(n.IsKind)))
                 .Select(n => SideEffectExpressions.Single(s => s.Kinds.Any(n.IsKind)).AffectedExpression(n));
+
+        private static ISymbol TupleArgumentSymbolMatchingLoopCounter(TupleExpressionSyntaxWrapper expression,
```

Comment:
I'm not really happy with `TupleArgumentSymbolMatchingLoopCounter` - any suggestion welcome.

Reply:
Not relevant anymore.